js/forum/src/components/EditUserModal.js

```diff
@@ -100,6 +111,24 @@ export default class EditUserModal extends Modal {
     );
   }
 
+  activate() {
+    this.loading = true;
+    const data = {
+      username: this.username(),
+      isActivated: true,
+    };
+    this.props.user.save(data, {errorHandler: this.onerror.bind(this)})
+      .then(() => {
+        this.isActivated(true);
+        this.loading = false;
+        m.redraw();
+      })
```

Comment:
I believe this should set the user property to the returned object, instead of overwriting the `isActivated` property, correct, @tobscure ?